acceptance/README.md

```diff
@@ -61,6 +61,39 @@ The following custom environment variables are made available to the scripts:
  * `HOME`: Set to the initial working directory. Required for `git` operations
  * `GH_CONFIG_DIR`: Set to the initial working directory. Required for `gh` operations
 
+#### Custom Commands
+
+The following custom commands are defined within [`acceptance_test.go`](./acceptance_test.go) to help with writing tests:
+
+- `defer`: register a command to run after the testscript completes
+
+  ```txtar
+  # Defer repo cleanup
+  defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+  ```
+
+- `env2lower`: set environment variable to the lowercase version of another environment variable
```

Comment:
**Question**

Is this used anywhere? Is there somewhere we expect it to be used?

If not...

**Change Requested**

Please remove it.

Reply:
I assume this request is because you value only having code around that is actively used.  I value having tools readily at hand for various scenarios.  One such scenario I can foresee but haven't written the test to require is asserting the repository name, which is always lowercase.

If there was a way to handle dynamic, case insensitive comparison where this wasn't needed, I'd get rid of both custom commands all together.

Reply:
_Subscribe_ 😄 

(I'm interested in this discussion and want to see where it goes, so jumping on the thread via this comment so I'll get notifications about it)

Reply:
> I assume this request is because you value only having code around that is actively used.

Actively used or at least with the intent to use, which is why I asked whether you expected it to be used somewhere.

Butttt it's a little more nuanced than whether the code is actively used or not.

Mostly I'm interested in avoiding a proliferation of commands. With each new command we add, the larger potential variation in the tests we have to maintain. I think new commands should have a fairly high bar for inclusion and should be supported by their usage. I'm keen to avoid a situation where commands are added because they might be useful, and then because they exist they get used.

All that said, if you have a plan of where it's going to be used and you don't want to wait until then to include it, I'm not going to use any more virtual ink on the subject.